### PR TITLE
LPD-39678 Performance issues deleting object entries

### PR DESCRIPTION
**Pull request by pedro-oliveira446**
[LPD-39678](https://liferay.atlassian.net/browse/LPD-39678)

[LPD-39678]: https://liferay.atlassian.net/browse/LPD-39678?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: modules/apps/object/object-service/src/main/java/com/liferay/object/internal/related/models/ObjectEntry1toMObjectRelatedModelsProviderImpl.java [hunk 1]
```diff
@@ -68,9 +77,40 @@ public void deleteRelatedModel(
 				deletionType,
 				ObjectRelationshipConstants.DELETION_TYPE_CASCADE)) {
 
+			ExecutorService executorService = new ThreadPoolExecutor(10, 10);
```
Comment (linolaoj) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/internal/related/models/ObjectEntry1toMObjectRelatedModelsProviderImpl.java:80]: Hi @pedro-oliveira446, can we use the 
`ExecutorService executorService =
			SystemExecutorServiceUtil.getExecutorService();` ?
On SystemExecutorServiceUtil, the poolSize and coreSize is already configured.

**Review by linolaoj — CHANGES_REQUESTED** [review]

File: modules/apps/object/object-service/src/main/java/com/liferay/object/internal/related/models/ObjectEntry1toMObjectRelatedModelsProviderImpl.java (hunk 1 above)
Comment (pedro-oliveira446) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/internal/related/models/ObjectEntry1toMObjectRelatedModelsProviderImpl.java:80]: Hey @linolaoj,

In my local environment, every time I tested it with the current code, it worked as expected. I attempted to implement the change you requested, but the results were inconsistent, and I wasn't able to pinpoint the exact cause.

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

**Comment by pedro-oliveira446** [comment]
ci:test:object

**Review by linolaoj — APPROVED** [review]

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

**Comment by pedro-oliveira446** [comment]
ci:test:object

**Review by linolaoj — APPROVED** [review]

**Comment by pedro-oliveira446** [comment]
ci:report:50311256

**Comment by pedro-oliveira446** [comment]
Resent https://github.com/liferay-bpm/liferay-portal/pull/3812

**Pull request closed without merging**